fronts-client/src/components/form/PaletteForm.tsx

```diff
@@ -0,0 +1,197 @@
+import React, { useCallback } from 'react';
+import { noop } from 'lodash';
+
+import {
+  ChefPalette,
+  ChefPaletteId,
+  CustomPalette,
+  chefPalettes,
+} from 'constants/feastPalettes';
+import { styled } from 'constants/theme';
```

Comment:
Noob Q: is `PaletteForm.tsx` unique for Chef-based work? Do we imagine it being more widely adopted later? Just wondering about the naming convention really.

Reply:
I think parts of this file will be used by the Collection palette work, but Collection palettes have a slightly different format – they have pictures, and a light and dark palette.

So, suspect that we'll have two forms here, and some of the components contained in this file will be split between them, if that makes sense.

Happy to change something if we reckon there might be a better name here.